kwset.c

```diff
@@ -41,6 +41,14 @@
 #define obstack_chunk_alloc xmalloc
 #define obstack_chunk_free free
 
+#ifdef USE_MSVC_CRTDBG
+#undef xmalloc
+static void *xmalloc(size_t size)
+{
+	return crtdbg_xmalloc(size, "kwset.c", -1);
+}
+#endif
+
```

Comment:
Would it work if we simply renamed all the `obstack_chunk_*()` functions instead? Like, do away with the funny `obstack_chunk_free()` and use, uhm, `free()` instead...?

Reply:
I did that as a hack because of the obstack alloc prototype and the vtable-like thing.  See _obstack_begin() in compat/obstack.c and the obstack_begin() macro in obstack.h.  They
in effect set the chunkfun structure field to xmalloc.  It looked like this would get really
messy, so I opted to just skip detailed tracing for these allocations.

Reply:
Okay...